### PR TITLE
[devtools] Add DevTools footer message

### LINKED ISSUE #835: Display + link to blog post tags from blog post detail pages
**Is your feature request related to a problem? Please describe.**

Whenever I’m reading a “what’s new in DevTools” blog post like https://developer.chrome.com/blog/new-in-devtools-89/, I want to click through to read other posts in this series.

We have several pages for that:

- https://developer.chrome.com/tags/new-in-devtools/ (the tag page)
- https://developer.chrome.com/docs/devtools/updates/ (the DevTools-specific overview page)

However, the blog post template doesn't link to any of these.

**Describe the solution you'd like**
It'd be nice if blog post tags were displayed + linkified on blog post detail pages. This would address the problem in a generic way, that's useful beyond just the DevTools posts.

Alternatively, we could add a link to https://developer.chrome.com/docs/devtools/updates/ to blog post detail pages if and only if the blog post has the “what’s new in DevTools” tag.

### PR DESCRIPTION
**Pull request by chybie**
Fixes #835 #842

Changes proposed in this pull request:

- Add a partial for Chrome preview channels and contact us message
- Add a partial for a list of what's new in DevTools features
- Update What's new in DevTools link in doc to point to tag page
- Update post 89-93 to include the partials

Screen Shot 2021-08-24 at 6 17 41 PM

### CODE REVIEW

File: site/_data/docs/devtools/toc.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 - url: /docs/devtools/overview/
 - url: /docs/devtools/open/
-- url: /docs/devtools/updates/
 # Because this page is generated by 11ty pagination, we need to give it an
```
Comment (samthor) [inline site/_data/docs/devtools/toc.yml:3]: this => "these" (since it now applies to several)

File: site/_data/docs/devtools/toc.yml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 - url: /docs/devtools/overview/
 - url: /docs/devtools/open/
-- url: /docs/devtools/updates/
```
Comment (samthor) [inline site/_data/docs/devtools/toc.yml:3]: Do you want to add an entry to redirects.yaml to go from this old URL to the tag page?

**Review by samthor — APPROVED** [review]
You're only starting from 89? Not adding the footer to all historic pages? 😆

**Comment by chybie** [comment]
> You're only starting from 89? Not adding the footer to all historic pages? 😆

I made it easier for the reviewer. 😝  Added all pages now!

**Review by samthor — APPROVED** [review]

**Pull request merged**